02-oop-inheritance/other/2_1/script.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by MaximilianoDaniel on 15/09/2015.
+ */
+var Movie = function () {
+    this.hashMap = "";
```

Comment:
**Minor:** I know it's an early comment, but just so you can apply it sooner than later: single quotes! :P